### DISCUSSION TITLE
Faster Startup Times

### DISCUSSION THREAD

**Discussion by R00tB33rMan** [AnOpenSauceDev/FastRandom#2] [General]
Mod appears to improve startup times on the launcher massively. Is this what you have also gathered from testing?

**Comment by AnOpenSauceDev** [comment]
I did not expect that! I have a few ideas as to how that *might* happen, but i need to test it first.

**Comment by AnOpenSauceDev** [comment]
@R00tB33rMan 
So i just tested the load times, and fastrandom made Minecraft load 2X faster! This is actually surprising and I didn't expect this to happen.

I might need to rebrand fastrandom...

**Reply by R00tB33rMan** [reply]
That’s awesome! Had a feeling it had a significant benefit. I was very pleased to see this simple yet effective piece of code work well. I’d encourage keeping it separate from your main mod, since it’s providing more than just an FPS boost and zero incompatibilities and crashes, across the board
